operatoringress/v1/types.go

```diff
@@ -55,6 +58,17 @@ type DNSRecordSpec struct {
 	// +kubebuilder:validation:Minimum=0
 	// +required
 	RecordTTL int64 `json:"recordTTL"`
+	// dnsManagementPolicy denotes the current policy applied on the dns record.
+	// Records that have policy set as "Unmanaged" are ignored by the ingress
+	// operator and the DNS record on the cloud provider is not managed by the
```

Comment:
When set to unmanaged, what happens to the Status?  Keeping stale status would be a problem.

Reply:
If an ingresscontroller is created with `Unmanaged` policy, the `status` is kept empty, so there no question of this being stale. When you update the `dnsManagementPolicy` to `Unmanaged`, the `status` from the previous state is retained as is, the `status` kind of has a consolidated view of all the necessary  information that the user/admin needs to manually create DNS records externally and also to perform a clean up of the DNS zone and DNS records that were provisioned on the cloud (we do not delete the cloud resources when set to `Unmanaged`). This was kind of the main reason we decided to retain the previous `status`, without which it could potentially leave the cloud DNS resources orphaned. 

I'm not entirely sure we could call `status` as stale because it still holds valid information regarding the DNS records, etc that are allocated on the cloud (at least until manual deletion on the cloud).

Reply:
> I'm not entirely sure we could call `status` as stale because it still holds valid information regarding the DNS records, etc that are allocated on the cloud (at least until manual deletion on the cloud).

This isn't the case. If you monitored to see what tags existed, it would be true.  Similarly if the conditions were updated, this would be true.  But near as I can tell looking at the API, only the ID will remain correct.  You cannot reasonably keep stale status.  If you don't observe the status, you don't even know if the ID still exists.